### DISCUSSION TITLE
Trace doctype usage.

### DISCUSSION THREAD

**Discussion by Ekerothh** [umbraco/Umbraco-CMS#14771] [Features and ideas]
Would be very helpful to be able to trace where an element or a documenttype is being used for composition. I.e check the compisition tab while inspecting an element/documenttype and there be able to see that it is being used at x, y and z locations.

**Comment by nul800sebastiaan** [comment]
Just as an FYI you could find this information now, using the excellent Diplo God Mode package: https://marketplace.umbraco.com/package/diplo.godmode

**Reply by marcemarc** [reply]
I know this is counter intuitive but if you are on a Document Type used for a composition, and you click to try to add a Composition to it... Which you can't do... It tells you where its currently used! But GodMode is super cool too.
